Source/WebGPU/WGSL/AST/ASTTypeDecl.h

```diff
@@ -143,7 +157,42 @@ class ParameterizedType : public TypeDecl {
     UniqueRef<TypeDecl> m_elementType;
 };
 
-} // namespace WGSL::AST
+class StructType final : public TypeDecl {
```

Comment:
Does it really make sense to have all these classes jammed together in the same file?

Reply:
I have a refactoring where I split them out. I should get to landing all that stuff.